.latch/workflow_name

```diff
@@ -0,0 +1 @@
+nf_nf_core_ampliseq
```

Comment:
maybe change name to nf_core_ampliseq (up to you, doesn't really matter)?

Reply:
we had hardcoded this to console already for the nfcore graphic that Nathan made so it was too late